### PR TITLE
Fix - Popover component import issue

### PR DESCRIPTION
**Pull request by dgiraldo313**
Took the time to refactor some of the folder structure in the popover component.

Although I'm still not able to import the component explicitly, I was able to imported on to external apps using `import Popover from 'pcln-popover/dist/index.js'`

### CODE REVIEW

File: packages/popover/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# pcln-popver
+# pcln-popover
```
Comment (sdalonzo) [inline packages/popover/README.md]: 👍

File: packages/popover/README.md [hunk 2]
```diff
@@ -7,25 +7,26 @@ npm i pcln-popover
 ```
 
 ```jsx
-import Popover from 'pcln-popover'
+// TODO: triage explicit import path
+import Popover from 'pcln-popover/dist/index.js'
```
Comment (sdalonzo) [inline packages/popover/README.md]: make sure this gets opened as an issue, not just a TODO in the codeline

File: packages/popover/README.md [hunk 3]
```diff
@@ -7,25 +7,26 @@ npm i pcln-popover
...
-  ariaLabel='Test PopOver'
-  overlayOpacity={0.3}
-  width={350}
-  bg='lightBlue'
-  borderColor='borderGray'
-  idx={1}
->
-  <Button>Popover</Button>
-</PopOver>
+  <Popover
```
Comment (sdalonzo) [inline packages/popover/README.md]: 👍

File: packages/popover/src/PopoverContent.js [hunk 4]
```diff
@@ -4,9 +4,9 @@ import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import { Popper } from 'react-popper'
 import { Box, theme } from 'pcln-design-system'
-import DEFAULTS_MODIFIERS from '../helpers/defaultModifiers'
-import Overlay from '../overlay'
-import PopoverArrow from '../arrow'
+import DEFAULTS_MODIFIERS from './helpers/defaultModifiers'
+import Overlay from './Overlay'
```
Comment (sdalonzo) [inline packages/popover/src/PopoverContent.js]: much better

File: packages/popover/test/Overlay.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { render, fireEvent, cleanup } from 'react-testing-library'
-import Overlay from '../src/overlay'
+import Overlay from '../src/Overlay.js'
```
Comment (sdalonzo) [inline packages/popover/test/Overlay.js]: i hope the `.js` isn't necessary

File: packages/popover/README.md (hunk 2 above)
Comment (BeniCheni) [reply packages/popover/README.md]: #504 has been created to follow up w. this `TODO` comment. (cc @sdalonzo )

File: packages/popover/test/Overlay.js (hunk 5 above)
Comment (BeniCheni) [inline packages/popover/test/Overlay.js]: @dgiraldo313 & @sdalonzo, I've used the suggested change feature to omit `.js`. Once tested locally, I will apply the change to this thread and move forward with the merge.

```suggestion
import Overlay from '../src/Overlay'
```

**Pull request merged**